src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/eltwise.cpp

```diff
@@ -66,10 +66,18 @@ ov::Tensor EltwiseLayerCPUTest::generate_eltwise_input(const ov::element::Type&
     } else {
         switch (type) {
             case ov::element::i8:
-                params = gen_params(INT8_MAX, INT8_MIN);
+                if (adopt_intervals) {
+                    params = gen_params(11 * 2, -11);
```

Comment:
Why these values are choosen? Maybe we can add some comment that will describe why range narrowing is needed

Reply:
comment was added, please review